hostmetadata/agent/agent.go

```diff
@@ -76,5 +71,4 @@ func AddMetadata(result map[string]string) {
 		strconv.FormatUint(uint64(config.ProbabilisticThreshold()), 10)
 	result[keyAgentConfigPresentCPUCores] =
 		strconv.FormatUint(uint64(config.PresentCPUCores()), 10)
-	result[keyAgentEnvHTTPSProxy] = os.Getenv("HTTPS_PROXY")
```

Comment:
This removal seems unrelated to the field names removal. Should it be its own PR?

Reply:
That's fine to me. Added the code back into the PR.